### PR TITLE
[line] do not show 0 or nulls in rich tooltip

### PR DESCRIPTION
**Pull request by mistercrunch**
screen shot 2018-04-09 at 11 10 17 pm

👀 @betodealmeida

### CODE REVIEW

File: superset/assets/visualizations/nvd3_vis.js [hunk 1]
```diff
@@ -424,17 +424,24 @@ export default function nvd3Vis(slice, payload) {
             + '</td></tr></thead><tbody>';
           d.series.sort((a, b) => a.value >= b.value ? -1 : 1);
           d.series.forEach((series) => {
```
Comment (betodealmeida) [inline superset/assets/visualizations/nvd3_vis.js]: Might be cleaner/clearer to rewrite this to:

```javascript
d.series.filter(series => series.value).forEach((series) => {
...
});
```

**Review by betodealmeida — COMMENTED** [review]
I'm worried about filtering out zeros here (I'm fine with `null`s). What if we're looking at metrics where zeros are relevant? Say, rides per region, and we had a region with zero rides?

**Comment by mistercrunch** [comment]
@vylc what do you think about `0`s ?

**Comment by betodealmeida** [comment]
> Default off/hide as well

But showing them is currently not an option — I'm fine with it being an option.

**Comment by mistercrunch** [comment]
@vylc  `Show Zeroes in Tooltip` boolean?

**Pull request closed without merging**